moda-contracts/src/interfaces/Catalog/ICatalogName.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+interface ICatalogName {
```

Comment:
I would say we should make this interface `IName` if `name()` were a common interface across multiple contracts. the I`Catalog`Name part doen't make sense anymore (There is nothing Catalog-y about `name()`) and we could simplify this to make it reusable. We now have the `Profile` contract, however, which supports `name` in the metadata for ERC721 along with many more fields including description, image, etc. I think we can drop this from the code itself and rely on Profile SBTs to provide this info.